aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,53 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## November 2022
+
+- **Avatar**
+  - We've added Avatar [guidance and examples](https://design-system.hpe.design/components/avatar) and [Figma component](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1&t=bQK90xgXcu9vlYYm-0).
```

Comment:
For smoother page transitions ---- For internal URLs, NextJS will trigger a true page reload when provided a complete URL; it will do a render without page reload when provided just the path.

```suggestion
  - We've added Avatar [guidance and examples](/components/avatar) and [Figma component](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1&t=bQK90xgXcu9vlYYm-0).
```